pkg/config/config.go

```diff
@@ -36,9 +36,9 @@ var (
 	ValidatingWebhookServicePath       = "/validate"
 	PolicyValidatingWebhookServicePath = "/policyvalidate"
 	PolicyMutatingWebhookServicePath   = "/policymutate"
-	KubePolicyAppLabels                = map[string]string{
-		"app": "kyverno",
-	}
+	// KubePolicyAppLabels                = map[string]string{
```

Comment:
Can we just remove this if it is not used?

Reply:
removed